src/composables/queries/useVeBalLockInfoQuery.ts

```diff
@@ -29,11 +28,6 @@ export default function useVeBalQuery(
    */
   const enabled = computed(() => isWalletReady.value && isVeBalSupported.value);
 
-  /**
-   * QUERY INPUTS
-   */
-  const queryKey = reactive(QUERY_KEYS.Tokens.VeBAL(networkId, account));
-
```

Comment:
These query keys are not actually reactive, since it is a function they lose reactivity åfter the initial return. I do not recommend this methodology anymore

Reply:
True, makes sense.

Reply:
What if the `reactive` will be inside the return of the function? That should solve it no?

Reply:
Ill test that out

Reply:
@evgenyboxer nah doesn't work